main.go

```diff
@@ -24,8 +24,15 @@ import (
 
 const listenAddr = ":9990"
 
+func defaultConfigPath() string {
+	if os.Getenv("GOKRAZY_FIRST_START") == "1" {
```

Comment:
Assuming that running `gibot -c /perm/gibot.json` is not an option within gokrazy, please document here why.

Reply:
I see. I still think a line or two here would help me personally, because otherwise I might forget about this exchange and think "wait, why is this here to begin with?" in a few years.

The comment doesn't need to go into specifics. Something like `Make it easy for gokrazy users to run this main package with no changes` would be enough to ensure that this special treatment isn't removed in the future.

Reply:
Done.